src/ui/layout/qgslayoutmapwidgetbase.ui

```diff
@@ -752,6 +752,20 @@
               </property>
              </widget>
             </item>
+            <item>
+             <widget class="QPushButton" name="mCopyGridPushButton">
+              <property name="toolTip">
+               <string>Duplicate selected grid</string>
+              </property>
+              <property name="text">
+               <string/>
+              </property>
+              <property name="icon">
+               <iconset resource="../../../images/images.qrc">
+                <normaloff>:/images/themes/default/mActionEditCopy.svg</normaloff>:/images/themes/default/mActionEditCopy.svg</iconset>
```

Comment:
mActionDuplicateLayer.svg is a better fit here -- that's what we use for eg the "duplicate symbol layer" button.